src/modules/wallet/components/RequestSignStatus/index.js

```diff
@@ -29,12 +29,21 @@ const successData = (t) => ({
   ),
 });
 
+const getStringifiedTransactionJSON = (signedTransaction) => {
+  const moduleCommand = joinModuleAndCommand(signedTransaction);
+  const paramSchema = moduleCommandSchemas[moduleCommand];
+  const transactionJSON = toTransactionJSON(signedTransaction, paramSchema);
+  return JSON.stringify(transactionJSON);
+};
```

Comment:
I think it would be best to get `moduleCommandSchemas` from `useCommandSchema` hook.

Reply:
Do you think it is safe? This above code is how we did it previously. I just moved it out of the component. 
<img width="838" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/f4899873-cacf-474a-adad-5a69fbb73861">

I am worried about loading state if we introduce that hook, could affect this component  🤔

Reply:
hmmm okay